### PR TITLE
implement twitter bootstrap and remake simple form for it

### PR DESCRIPTION
**Pull request by kont-noor**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,9 +44,6 @@ gem "bitmask_attributes"
 # imagemagick support
 gem "mini_magick", "~> 3.4"
 
-# pagination
-gem "kaminari", "~> 0.14.1"
```
Comment (DevMike) [inline Gemfile:45]: are you sure? it seems it used for activeadmin

File: app/views/devise/registrations/new.html.haml [hunk 2]
```diff
@@ -8,10 +8,13 @@
     - [:country, :region, :settlement].each do |location|
       = render :partial => 'locations/input', :locals => {:f => f, :resource => resource, :location => location}
     = f.input :agree, :required => true, :input_html => {:type => 'checkbox'}
-  .actions
-    = f.button :submit, t(:"users.register")
+  .form-actions
+    = f.button :submit, t(:"users.register"), class: 'btn'
 = render :partial => "devise/shared/links"
 
+
```
Comment (DevMike) [inline app/views/devise/registrations/new.html.haml:15]: don't see sense for the empty lines

File: config/locales/simple_form.en.yml [hunk 3]
```diff
@@ -9,16 +9,18 @@ en:
...
-    #       email: 'E-mail para efetuar o sign in.'
+    #       email: 'E-mail to sign in.'
     #     edit:
     #       email: 'E-mail.'
     # hints:
-    #   username: 'User name to sign in.'
-    #   password: 'No special characters, please.'
+    #   defaults:
+    #     username: 'User name to sign in.'
+    #     password: 'No special characters, please.'
```
Comment (DevMike) [inline config/locales/simple_form.en.yml:25]: are there any ones for russian version?

File: app/views/devise/registrations/new.html.haml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-= simple_form_for(resource, :as => resource_name, :url => registration_path(resource_name)) do |f|
+= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: {class: 'well form-horizontal'}) do |f|
```
Comment (DevMike) [inline app/views/devise/registrations/new.html.haml:1]: is there any way to attach appropriated class to all forms by default?

File: Gemfile (hunk 1 above)
Comment (kont-noor) [inline Gemfile:45]: yep.
activeadmin installs it by itself

**Pull request merged**